classes/form/CustomerAddressFormatter.php

```diff
@@ -146,6 +146,21 @@ public function getFormat()
             $format[$formField->getName()] = $formField;
         }
 
+        //To add the extra fields in address form
+        $additionalAddressFormFields = Hook::exec('additionalCustomerAddressFields', array(), null, true);
```

Comment:
what if 2 modules calls the same hook?

Reply:
@mickaelandrieu 
If 2 modules call the same hook and try to add an extra field then the field from both module will be added.
Both modules will work without any confliction.
It will work as like as a hook on customer registration form(additionalCustomerFormFields).